src/analyses/apron/apronPriv.apron.ml

```diff
@@ -579,6 +579,8 @@ struct
   let threadenter ask getg (st: apron_components_t): apron_components_t =
     {apr = AD.bot (); priv = startstate ()}
 
+  let invariant_vars ask getg st = protected_vars ask
+
```

Comment:
Regarding this and the following comments: this still needs to be done, because if it were `[]`, then no globals would ever be in invariants. This is for all globals to include in invariants, not extra globals to include/exclude in addition to the ones in the local state, because it could be both ways depending on the privatization: the local state may contain more or less globals.

Reply:
I now implemented a slight variation for mutex-meets, which gets the list of protected variables directly from the local state, rather than with all the queries.